pages/index.js

```diff
@@ -5,14 +5,25 @@ import {Button} from '../components/styled-links'
 import Layout from '../components/layout'
 import Separator from '../components/separator'
 import CodePreview from '../components/code-preview'
+import MarkdownWrapper from '../components/markdown-wrapper'
 
 const {Div} = glamorous
 
+// Quick and Dirty copy paste
+
+// Define main file here so we can require the markdown file in the client
+let main = ''
+
+// Adds in main.md on client only
+if (typeof window !== 'undefined' && window.__NEXT_DATA__ !== undefined) {
```

Comment:
`if (typeof window !== 'undefined') {`

We don't need to know if there's `__NEXT_DATA__` to require the .md files. I think this is a copy/paste issue I introduced a little while ago. 🙈